lib/l10n/app_en.arb

```diff
@@ -295,6 +295,8 @@
     "l_fido_pin_protection_optional": "Optional FIDO PIN protection",
     "l_enter_fido2_pin": "Enter the FIDO2 PIN for your YubiKey",
     "l_pin_blocked_reset": "PIN is blocked; factory reset the FIDO application",
+    "l_fido_pin_blocked": "PIN is blocked",
+    "l_note_attempts_remaining": "Note: {retries} attempt(s) remaining",
```

Comment:
Flutter has support for Plurals in localized strings: https://docs.flutter.dev/ui/accessibility-and-internationalization/internationalization#placeholders-plurals-and-selects

Reply:
Sounds interesting! There's definitely a lot of strings we could use that for. I will investigate this and hopefully incorporate it in a future PR.